server/cherry_pick.go

```diff
@@ -56,14 +56,14 @@ func (s *Server) checkIfNeedCherryPick(pr *model.PullRequest) {
 	ctx, cancel := context.WithTimeout(context.Background(), defaultRequestTimeout*time.Second)
 	defer cancel()
 
-	prCherryCandidate, _, err := s.GithubClient.PullRequests.Get(ctx, pr.RepoOwner, pr.RepoName, pr.Number)
-	if err != nil {
-		mlog.Error("Error getting the PR info", mlog.Err(err))
+	if !pr.Merged.Valid || !pr.Merged.Bool {
```

Comment:
All that SQL abstraction is totally lost now :D I am wondering if a helper method like IsMerged to abstract away this `NullBool` logic makes sense?

Reply:
The extremist in me says lets just drop the DB ;)

The go-github library has getters like `pullRequest.GetMerged()`. We could model our model on their model :)